docs/develop/common-coding-issues.md

```diff
@@ -35,6 +41,17 @@ You can identify a property that must have its subproperties set with a JSON str
 - Read-only property: Subproperties can be set through navigation.
 - Writable property: Subproperties must be set with a JSON struct (and cannot be set through navigation).
 
+## Excel Range limits
+
+Excel developers need to be aware of two data size limitations:
+
+- Excel on the web has a payload size limit for requests and responses of 5MB. `RichAPI.Error` will be thrown if that limit is exceeded.
+- A range is limited to five million cells for set operations.
+
+If you expect user input to exceed those amounts, be sure to check the data and split the ranges into multiple objects. You'll also need separate `context.sync()` calls to avoid overwhelming the Excel cloud service.
```

Comment:
```suggestion
If you expect user input to exceed these limits, be sure to check the data and split the ranges into multiple objects. You'll also need to submit multiple `context.sync()` calls to optimize performance.
```

Reply:
I'm taking the first two changes. To the third point, the issue isn't performance. They'll need to use multiple syncs to avoid the size limit error. I'll try to highlight that point better.